config/cmake/modules/FindHYPRE.cmake

```diff
@@ -28,6 +31,55 @@ if (HYPRE_FOUND)
   return()
 endif()
 
+if (HYPRE_FETCH OR FETCH_TPLS)
+  set(HYPRE_FETCH_VERSION 2.32.0)
```

Comment:
```suggestion
  set(HYPRE_FETCH_VERSION 2.33.0)
```

Reply:
should be addressed in https://github.com/mfem/mfem/pull/4673/commits/01eeccee7f5e3d65e5d85308726076fdc72a9907